docs/getting-started/create-first-cluster/using_fleet.md

```diff
@@ -2,6 +2,133 @@
 sidebar_position: 3
 ---
 
-# Using fleet
+# Create & import your first cluster using Fleet
 
-This section needs to be filled out.
\ No newline at end of file
+This section will guide you through creating your first cluster and importing it into Rancher Manager using a GitOps workflow with Fleet.
+
+## Prerequisites
+
+- Rancher Manager cluster with Rancher Turtles installed
+- Cluster API providers installed for your scenario - we'll be using the Docker infrastructure and Kubeadm bootstrap/controlplane providers in these instructions
+- **clusterctl** CLI - see the [releases](https://github.com/kubernetes-sigs/cluster-api/releases)
+
+## Create your cluster definition
+
+The **clusterctl** CLI can be used to generate the YAML for a cluster. When you run `clusterctl generate cluster` command is run, it will connect to the management cluster to see what infrastructure providers have been installed. Also, it will take care of replacing any tokens in the chosen template (a.k.a flavour) with values from environment variables.
```

Comment:
is the part of figuring out what infrastructure providers are installed true in case of using CAPI operator?

Reply:
Yes i believe it is.